### DISCUSSION TITLE
Support multimatch pipeline script

### DISCUSSION THREAD

**Discussion by fz6m** [vercel/turborepo#377] [Ideas]
### Describe the feature you'd like to request

Many times there are multiple scripts with different purposes, but they only differ in the suffix:

```js
  "scripts": {
    "test:ci": "...",
    "test:local": "...",
    "test:prod": "...",
     // ...
  },
```

But they all need to be configured:

```js
  "turbo": {
    "pipeline": {
      "test:ci": {
        "outputs": []
      },
      "test:local": {
        "outputs": []
      },
      "test:prod": {
        "outputs": []
      },
      // .....
    }
  }
```

### Describe the solution you'd like

Support for multimatch with script suffix:

```js
  "turbo": {
    "pipeline": {
      "test:*": {
        "outputs": []
      }
    }
  }
```

### Describe alternatives you've considered

It seems to be difficult to achieve auto config

**Comment by anthonyshew** [comment]
The ideal is that you wouldn't need to have these forked scripts anymore. The goal is for us to create consistent pipelines for all environments through `turbo`, so this would probably end up being a mis-feature long term.
